roles/hosts/defaults/main.yml

```diff
@@ -3,10 +3,6 @@
 # or groups of hosts in the hosts file.
 hosts_enable: true
 
-# The IPv4 address assigned to this interface is placed in the
-# hosts file.
-hosts_interface: "{{ management_interface|default('lo') }}"
```

Comment:
Comment that and remove ``"{{ management_interface|default('lo') }}"``.

I want to keep the description and the commented parameter itself here for documentation.

Reply:
Has been fixed!